docs/getting-started/sdk/internal/index.md

```diff
@@ -32,11 +32,15 @@ See the [Tools and Libraries](../../tools/index.md) page for more information.
 
 ## Building the SDK
 
-To build the SDK, run the following command:
+The SDK is built for different platforms, all of which have their own build instructions. For more
+information on how to for a specific platform, refer to the readmes for the different crates:
 
-```bash
-cargo build
-```
+- **Web**: `crates/bitwarden-wasm-internal`
+- **iOS**: `crates/bitwarden-uniffi/swift`
+- **Android**: `crates/bitwarden-uniffi/kotlin`
+
+If you encounter any issues, please check the readme and make sure that you have installed all of
+the required dependencies for that platform.
```

Comment:
This seems a bit repetitive?

Reply:
Hmm, I changed the section a little bit to focus on what I'm actually trying to draw attention to